### ISSUE TITLE
In SolidFire 12.2 or later Max iSCSI sessions may be impacted by node's MaintenanceMode 

### ISSUE THREAD

**Issue by scaleoutsean** [scaleoutsean/nagfire#1]
The API guide isnt' clear on how to `Get` node mode but there's `EnableMaintenanceMode` and `DisableMaintenanceMode`; the former takes the node out of service and effectively removes the ability to host volumes and iSCSI sessions.
The Max iSCSI sessions formula will need more changes similar to those in made in nagfire v2.1 for two storage node clusters.

Documentation:

https://docs.netapp.com/sfe-122/topic/com.netapp.doc.sfe-api/GUID-9A224B03-75F9-4288-8F2A-3E2BAD5918B0.html?resultof=%22%6d%61%69%6e%74%65%6e%61%6e%63%65%6d%6f%64%65%22%20%22%6d%61%69%6e%74%65%6e%61%6e%63%65%6d%6f%64%22%20

**Comment by scaleoutsean** [comment]
It seems ListActiveNodes (also used to accommodate Witness Nodes in v12.1) can give us the info we need. Next thing is to get sample output from a disabled node.

**Comment by scaleoutsean** [comment]
```raw
# it's ListAllNodes
$resp.nodes[0].maintenanceMode
Disabled
```

There's no direct way to get maintenance mode - we'd have to find how many have `maintenanceMode: Enabled`.

12.3 docs link: https://docs.netapp.com/us-en/element-software/esds/reference_esds_use_maintenance_mode.html#disable-maintenance-mode

**Comment by scaleoutsean** [comment]
Should be addressed in v2.2.

**Comment by scaleoutsean** [comment]
Should be finally fixed in commit a97cb5ed0b95d32f9df92906eb6cdb87d094db9c
